### PR TITLE
Add some ICEs

### PR DESCRIPTION
**Pull request by langston-barrett**

### CODE REVIEW

**Comment by langston-barrett** [comment]
CI failure is #1534.

**Comment by langston-barrett** [comment]
CI is fixed, added some more ICEs.

File: ices/109239.rs [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#![feature(return_position_impl_trait_in_trait)]
+#![crate_type = "lib"]
+
+trait Trait {
+    type Type;
+
+    fn method(&self) -> impl Trait<Type = impl Trait<Type = impl Sized + '_> + '_>;
+}
+
+fn main() {}
```
[the remark this answers is not in the document]
Comment (langston-barrett) [reply ices/109239.rs]: Whoops, missed that one! Thanks.

**Review by Alexendoo — APPROVED** [review]
Thank you!

**Pull request merged**